src/prefect/cli/deployment.py

```diff
@@ -516,12 +517,29 @@ async def run(
         None,
         "--start-at",
     ),
+    watch: bool = typer.Option(
+        None,
+        "--watch",
+        help="Whether to poll the flow run until a terminal state is reached.",
+    ),
+    watch_interval: int = typer.Option(
+        None,
+        "--watch-interval",
+        help="How often to poll the flow run for state changes (in seconds).",
+    ),
+    watch_timeout: int = typer.Option(
+        None, "--watch-timeout", help="Timeout for `--watch`."
+    ),
+    tags: List[str] = typer.Option(
+        [], "--tag", help="Tag(s) to be applied to flow run"
+    ),
```

Comment:
nit: I think we should move this above the watch options so it gets displayed higher in the help.

Reply:
updated in [41b0908](https://github.com/PrefectHQ/prefect/pull/11702/commits/41b0908e1584a749b29d19022992299358a51140)